src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -183,9 +183,11 @@ class PolicyHandler : public PolicyHandlerInterface,
                          const std::string& policy_app_id,
                          const std::string& hmi_level) OVERRIDE;
 
-#ifndef EXTERNAL_PROPRIETARY_MODE
+  /**
+   * @brief OnPTUTimeOut the callback which signals if PTU timeout occured
```

Comment:
@ydementieiev Suggested changes here: 
 - `the callback which signals if PTU timeout occured` -> `the callback which is performed when PTU timeout occurred`
 - Move description to iface

Reply:
@AByzhynar  update description [c04c115ce](https://github.com/smartdevicelink/sdl_core/commit/c04c115ced5d0ea1dfcc5bb53fc9c01d6d09a998)